engine/class_modules/priest/sc_priest.hpp

```diff
@@ -558,6 +561,15 @@ struct priest_t final : public player_t
     const spell_data_t* boon_of_the_ascended;
   } covenant;
 
+  enum living_shadow_action
```

Comment:
1. If you don't want to prepend priest_t:: all the time for the enum values just move the whole thing out of priest _t class ;)
2. Prefer using a enum class. You can also simplify the enum values since they are bound to its scope,.a call might eg become `living_shadow_action::spike`

Reply:
I'd still recommend using a class enum so they don't leak out of scope :-)

Reply:
I did make it an `enum class` insetad already, unless im misunderstanding what you mean? @scamille